### PR TITLE
add lyra-v2 fees

### PR DESCRIPTION
**Pull request by 0xdomrom**

### CODE REVIEW

**Comment by llamatester** [comment]
The lyra-v2.ts adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees lyra-v2.ts
    
    🦙 Running LYRA-V2.TS adapter 🦙
    ---------------------------------------------------
    Start Date:	Sun, 08 Dec 2024 00:00:00 GMT
    End Date:	Mon, 09 Dec 2024 00:00:00 GMT
    ---------------------------------------------------
    
    LYRA 👇
    Backfill start time: 1/11/2023
    NO METHODOLOGY SPECIFIED
    End timestamp: 1733702399 (2024-12-08T23:59:59.000Z)
    Daily fees: 12.67 k

File: fees/lyra-v2.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { getUniqStartOfTodayTimestamp } from "../helpers/getUniSubgraphVolume";
+import axios from "axios";
+
+async function getLyraV2Fees(timestamp: number) {
+  const cleanTimestamp = getUniqStartOfTodayTimestamp(
+    new Date(timestamp * 1000)
+  );
+  const cleanTimestampMs = cleanTimestamp * 1000;
+
+  const data = await axios.get(`https://api.lyra.finance/public/statistics?instrument_name=ALL&end_time=${cleanTimestampMs}`, {
```
Comment (0xngmi) [inline fees/lyra-v2.ts]: can we pass any timestamp here? or does it need to be at the start of the day?

File: fees/lyra-v2.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+async function getLyraV2Fees({endTimestamp}) {
+  const data = await axios.get(`https://api.lyra.finance/public/statistics?instrument_name=ALL&end_time=${endTimestamp*1e3}`, {
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+
+  return {
+    dailyFees: data.data.result.daily_fees.toString(),
```
Comment (waynebruce0x) [inline fees/lyra-v2.ts]: what does `data.data.result.daily_fees` represent? Is it midnight to midnight data?

**Comment by llamatester** [comment]
The lyra-v2.ts adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees lyra-v2.ts
    
    🦙 Running LYRA-V2.TS adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 10 Dec 2024 00:00:00 GMT
    End Date:	Wed, 11 Dec 2024 00:00:00 GMT
    ---------------------------------------------------
    
    LYRA 👇
    Backfill start time: 1/11/2023
    NO METHODOLOGY SPECIFIED
    End timestamp: 1733875199 (2024-12-10T23:59:59.000Z)
    Daily fees: 24.21 k

**Pull request merged**